src/pages/home/report/ReportActionsView.js

```diff
@@ -338,9 +336,7 @@ class ReportActionsView extends React.Component {
     updateSortedReportActions(reportActions) {
         this.sortedReportActions = _.chain(reportActions)
             .sortBy('sequenceNumber')
-            .filter(action => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU
-                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT
-                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED)
+            .filter(action => _.contains(_.values(CONST.REPORT.ACTIONS.TYPE), action.actionName))
```

Comment:
This doesn't seem like a safe change. We might have action types that should not be rendered in the future added to this list. Can we just add the new action to this list?

Reply:
```js
            .filter(action => action.actionName === CONST.REPORT.ACTIONS.TYPE.IOU
                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT
                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED
                    || action.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED)
```

Reply:
Yes, that makes sense. Will do!

Reply:
Thanks!

Reply:
Sounds good.